### DISCUSSION TITLE
May I ask what I need to do to add a hyperlink to the graphic？

### DISCUSSION THREAD

**Discussion by TianYuanTuPo** [gosling-lang/gosling.js#969] [Q&A]
For example, the small rectangles below this website have hyperlinks added to each one. https://gosling.js.org/?example=BAM_PILEUP

**Comment by sehilyi** [comment]
Hi @TianYuanTuPo,

If you mean the tooltip, you can enable it by specifying the `tooltip` option:

![Screenshot 2023-10-06 at 14 20 54](https://github.com/gosling-lang/gosling.js/assets/9922882/601e3272-a8e4-40a2-831e-8b425fc534ff)

```ts
"tooltip": [
  {"field": "start", "type": "genomic"},
  {"field": "end", "type": "genomic"},
  {"field": "insertSize", "type": "quantitative"},
  {"field": "svType", "type": "nominal"},
  {"field": "strand", "type": "nominal"},
  {"field": "numMates", "type": "quantitative"},
  {"field": "mateIds", "type": "nominal"}
],
```
